src/Simplex/Messaging/Agent.hs

```diff
@@ -553,9 +570,10 @@ subscribeConnection' c connId =
   where
     subscribe :: RcvQueue -> m ()
     subscribe rq = do
-      subscribeQueue c rq connId
+      r <- tryError $ subscribeQueue c rq
```

Comment:
won't this lead to creation of notification subscription even if `subscribeQueue` fails?